src/patternfly/components/Table/examples/table-compact-expandable-example.hbs

```diff
@@ -0,0 +1,87 @@
+{{#> table table--id="compact-expandable-table" table--grid="true" table--modifier="pf-m-compact pf-m-grid-md" table--expandable="true" table--attribute='aria-label="Compact expandable table example"'}}
```

Comment:
I wonder if we should design if compact expandable tables need to have the border modifier (once we merge your PR that adds them). 

This looks off, but it might just be me
<img width="988" alt="Screen Shot 2019-03-13 at 3 03 21 PM" src="https://user-images.githubusercontent.com/20118816/54307090-30a9db00-45a1-11e9-949c-73e2cf53b539.png">

Reply:
re: borders, I think so. Is what looks off the lack of a border? between the top row and expanded row?

Reply:
yeah exactly

Reply:
but I think its a question from design, do you have the design specs?

Reply:
No, @mceledonia just said that the left border should behave like the regular expanded table example, but I agree it does look weird. Also not sure about the shadow on the right of the expanded area. That shadow fits better when there is a top border.

FWIW, here's what the compact table looks like in a better example with collapsed/expanded rows.

<img width="849" alt="Screen Shot 2019-03-13 at 3 06 01 PM" src="https://user-images.githubusercontent.com/35148959/54310970-a57d1500-45a1-11e9-9625-d3849255a6fa.png">